backup/setup/schools.json

```diff
@@ -338,5 +341,58 @@
 		"rssFeeds": [],
 		"__v": 0,
 		"purpose": "tombstone"
+	},
+	{
+		"_id": {
+			"$oid": "5f2987e020834114b8efd600"
+		},
+		"name": "OIDC-Mock-School",
+		"ldapSchoolIdentifier": "0000d186816abba584714c92",
+		"fileStorageType": "awsS3",
+		"federalState": {
+			"$oid": "0000b186816abba584714c53"
+		},
+		"county": {
+			"antaresKey": "BRB",
+			"_id": {
+				"$oid": "5fa55eb53f472a2d986c8812"
+			},
+			"countyId": 12051,
+			"name": "Brandenburg an der Havel"
+		},
+		"systems": [
+			{
```

Comment:
Do we need iserv and sanis for this school? I thought it is for the OIDC mock only

Reply:
Nope. This is a case of 'copy and waste' - Done